test/plugin/test_out_graphite.rb

```diff
@@ -94,6 +94,24 @@ def test_format_metrics
       'dstat.total cpu usage.siq' => '0.0'
     }
 
+    run_format_metrics_assertions(record)
+  end
+  
+  def test_format_metrics_symbols
+    record = {
+      :'hostname'                  => 'localhost.localdomain',
+      :'dstat.total cpu usage.usr' => '0.0',
+      :'dstat.total cpu usage.sys' => '0.0',
+      :'dstat.total cpu usage.idl' => '100.0',
+      :'dstat.total cpu usage.wai' => '0.0',
+      :'dstat.total cpu usage.hiq' => '0.0',
+      :'dstat.total cpu usage.siq' => '0.0'
+    }
```

Comment:
Could you follow [this](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#hashes)?